notebooks/02.08-Sorting.ipynb

```diff
@@ -30,7 +30,7 @@
    "cell_type": "markdown",
```

Comment:
<p><strong>Silly sorting.</strong></p><p><br></p><pre class="ql-syntax" spellcheck="false">a = b + c
</pre><p><br></p><p>Is that right?</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>